java/src/jmri/jmrix/powerline/SerialAddress.java

```diff
@@ -397,5 +421,34 @@ public int idLowCodeAsValueFromSystemName(String systemName) {
     }
 
     private final static Logger log = LoggerFactory.getLogger(SerialAddress.class);
+    
+    /**
+     * Extract DMX unit id from system name.
+     * <p>
+     * If the supplied system name does not have a valid format, an empty string
+     * is returned.
+     *
+     * @param systemName name
+     * @return dmx unit id, -1 if invalid
+     */
+
+    public int unitIdCodeAsValueFromSystemName(String systemName) {
```

Comment:
It might be good to rename this method to `dmxUnitIdCodeAsValueFromSystemName` to clarify that the method is for dmx.